src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+
+    private final int height;
+    private final int firstSide;
+    private final int secondSide;
+
+    public IsoscelesTrapezoid(Color color, int height, int firstSide, int secondSide) {
+        super(color);
+        this.height = height;
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure : IsoscelesTrapezoid; area : " + getArea() + " sq.units, "
+                + " first side : " + firstSide + " units, "
+                + " second side : " + secondSide + " units, "
+                + " height : " + height + " units, "
+                + " color : " + getColor());
```

Comment:
not fixed 
Fix this for all figures
```suggestion
                + " color : " + getColor().name());
```